### PR TITLE
Applied feedback from Equity Committee

### PR DESCRIPTION
**Pull request by jonahbron**

### CODE REVIEW

File: frontend/src/element/signup.ts [hunk 1]
```diff
@@ -369,6 +369,13 @@ export class Signup extends LitElement {
           complete: this.isComplete,
         })}"
       >
+        <p class="full-width">
+          If you have any trouble completing this form, you may instead contact
```
Comment (realjenny) [inline frontend/src/element/signup.ts:373]: Delete "you may instead"

File: frontend/src/element/signup.ts [hunk 2]
```diff
@@ -650,8 +657,9 @@ export class Signup extends LitElement {
             >Total Compensation (TC)${optionalLabel('total-compensation')}</span
           >
           <span class="hint"
-            >Annual. Used to calculate your union dues. We expect members to be
-            honest, but this is the honor system: we won't check.</span
+            >Used to calculate your union dues. Select the calculator icon if
```
Comment (realjenny) [inline frontend/src/element/signup.ts:660]: Just checking, what's the aria-label for the calculator icon? Can we make sure to use the same name?

**Review by realjenny — CHANGES_REQUESTED** [review]

**Review by butterflyhug — APPROVED** [review]

File: frontend/src/element/signup.ts (hunk 2 above)
Comment (jonahbron) [reply frontend/src/element/signup.ts:800]: "Compensation calculator"

File: frontend/src/element/signup.ts (hunk 1 above)
Comment (jonahbron) [reply frontend/src/element/signup.ts:491]: Done

**Pull request merged**